pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -777,3 +787,12 @@ func parseCommitMsg(msg string, args argsTemplate) string {
 	}
 	return buf.String()
 }
+
+// makeBranchName generates a new branch name in the format {eventName}-{uuid} if newBranch is true.
+// If newBranch is false, the function returns the existing branch name.
+func makeBranchName(newBranch bool, eventName, branch string) string {
+	if !newBranch {
+		return branch
+	}
```

Comment:
[nit] It would be nice to avoid using `!` when you can. 👀
we need to convert the value to not before understanding :)

Reply:
Thanks!
[avoid using '!'](https://github.com/pipe-cd/pipecd/pull/4395/commits/bbe6f49cd00f58cca0d8056ea9febedc6aba67de)